Rakefile

```diff
@@ -177,3 +182,23 @@ task :upgrade_test_descriptions do
     File.join(Machinery::ROOT, "spec/data/schema/validation_error/unmanaged_files")
   )
 end
+
+desc "Generate Machinery Test Matrix as a Spreadsheet"
+task "matrix:spreadsheet" do
+  begin
+    require_relative "tools/support_matrix/lib/support_matrix"
+  rescue LoadError => e
+    puts <<-EOF
+      Error: #{e.message}
+
+      You can solve this issue by:
+
+        1) running `bundle exec rake matrix:spreadsheet` or
+        2) by installing the gems binstubs `bundle install --binstubs`
+
+    EOF
+  end
+  sources = File.join(Machinery::ROOT, "spec", "definitions", "support")
+  file = SupportMatrix.new(sources, OdsFormatter.new).write(Machinery::DEFAULT_CONFIG_DIR)
```

Comment:
I wouldn't write the file to the config dir. It would make more sense to write it to the directory where the yaml files are stored as well. Maybe we should even check it in, so that it can be accessed when browsing the project in GitHub (and linked from the Wiki page).